modules/rational-ui.el

```diff
@@ -22,11 +22,24 @@ the same key names as accepted by `set-face-attribute'")
       doom-modeline-buffer-file-name-style 'truncate-except-project)
 
 ;; Make `describe-*' screens more helpful!
-(require 'helpful)
+(straight-use-package 'helpful)
```

Comment:
I don't think helpful is built-in, but I also don't use straight-use-package yet. Is this correct?

Reply:
I have it higher up in this file. The strategy I've been using so far is to put all the package ensure expressions at the top so its clearer which packages get installed by a module.